tests/integration/test_subscriptions_api.py

```diff
@@ -0,0 +1,177 @@
+"""Tests of the Planet Subscriptions API client."""
+
+import pytest
+
+import planet.clients.subscriptions
+from planet.clients.subscriptions import (PlaceholderSubscriptionsClient as
+                                          SubscriptionsClient)
+from planet.exceptions import ClientError
+from planet.http import Session
+
+
+@pytest.mark.xfail(reason="Client/server interaction not yet implemented")
```

Comment:
These tests are all marked xfail for the moment. We'll remove the markers in the next steps.